src/style.scss

```diff
@@ -231,13 +234,12 @@ slot {
       transform: unset;
 
       color: rgba($auro-color-text-link-on-light, $disabledOPacity);
-      background-color: transparent;
-      border: transparent;
+      background-color: $auro-rgb-color-base-black-03;
+      border: 1px solid transparent;
```

Comment:
Adding the `1px solid` here is problematic. Same is applied to the disabled hover style.

The problem is that there is no border size set when the button is not disabled. So applying disabled dynamically will cause the UI to jump as the box model changes.

The border 1px solid should be always present and the disabled state should just change the color. 

The --secondary style follows this pattern.

An even better way to write this would be to declare the border width and style globally to --tertiary and just change border color for each state.